modules/claims_api/spec/workers/report_unsuccessful_submissions_spec.rb

```diff
@@ -62,14 +74,22 @@
     it 'calculate totals' do
       with_settings(Settings.claims_api,
                     report_enabled: true) do
-        errored_upload_claims
+        errored_upload_claims.push(pending_claims)
```

Comment:
Not sure I understand why we're pushing the `pending_claims` into the `errored_upload_claims`.  🤔 
Do we need that?

Reply:
errored_uploaded_claims has been renamed to make more sense. THX